test/test_xla_dist.py

```diff
@@ -587,7 +587,7 @@ def test_unhealthy_pod_service_cluster(self):
             'state':
                 'READY',
             'health':
-                'UNHEALTHY_TENSORFLOW',
+                'UNHEALTHY_OPENXLA',
```

Comment:
I am curious if this flag continues to be useful in the openxla world?

Reply:
Thanks for pointing, will delete this `test/test_xla_dist.py` file since  this file is only for XRT

Reply:
Thanks, updated in https://github.com/pytorch/xla/pull/5202